src/api/Movies/Movies.api.ts

```diff
@@ -31,19 +33,25 @@ export const fetchMoviesAPI = async (
 
   return {
     ...data,
-    data: data.data.map(
-      (movie): Movie => ({
-        id: movie.id,
-        title: movie.title,
-        genre: movie.genres,
-        coverUrl: movie.poster_path,
-        description: movie.tagline,
-        rating: movie.vote_average,
-        releaseDate: movie.release_date,
-        runtime: movie.runtime,
-      })
-    ),
+    data: data.data.map((movie): Movie => movieApiToMovie(movie)),
   };
 };
 
 export type fetchMovieType = ReturnType<typeof fetchMoviesAPI>;
+
+export const updateMovie = async (movie: Movie) => {
```

Comment:
I'll repeat my own comment for previous PR: with axios instance these CRUD-methods could've been slightly more DRY.

Reply:
Do you have code examples of the approach you're recommending?
Would be helpful